packages/vite-plugin-cloudflare/playground/prisma/__tests__/prisma.spec.ts

```diff
@@ -9,7 +9,11 @@ import { getJsonResponse } from "../../__test-utils__";
 //
 // We do this in the `preServe()` hook, in `serve.ts`, that is called from the `packages/vite-plugin-cloudflare/playground/vitest-setup.ts` file.
 
-test("runs D1 query using Prisma", async () => {
+test.skipIf(
+	// for some reason this seems to be problematic on windows
+	// when running the various playground tests in parallel
+	process.platform === "win32"
+)("runs D1 query using Prisma", async () => {
```

Comment:
What if we just run the tests in sequence in Windows, rather than skipping this test?

Reply:
ah sure, that didn't occur to me! 🙂👍

Reply:
updated, thanks for the suggestion :slightly_smiling_face: